release-schema.json

```diff
@@ -182,6 +214,66 @@
           "title": "Buyer",
           "description": "The organization aiming to conclude a contract with a supplier or to use the goods, works or services in this lot. This may be different from the procuring entity who may be specified in the tender data.",
           "$ref": "#/definitions/OrganizationReference"
+        },
+        "additionalClassifications": {
+          "title": "Additional classifications",
+          "description": "An array of additional classifications for this lot.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Classification"
+          },
+          "uniqueItems": true,
+          "wholeListMerge": true
+        },
+        "mainProcurementCategory": {
+          "title": "Main procurement category",
+          "description": "The primary category describing the main object of this lot, from the closed [procurementCategory](https://standard.open-contracting.org/{{version}}/{{lang}}/schema/codelists/#procurement-category) codelist.",
```

Comment:
Extensions don't have {{version}} and {{lang}} substitution. Just name the codelist without the link. Occurs twice.

Reply:
Fixed in https://github.com/open-contracting-extensions/ocds_lots_extension/pull/41/commits/41a34f385959e3dd3f4ff0e6cf045622feec14d2